CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Cassava Megaparsec 2.1.0
```

Comment:
Hey @aemartinez37 I was thinking about the right version update and following the [Haskell pvp](https://pvp.haskell.org/) it seems this change is a minor release. 
> 8. Adding new modules. Adding new modules might cause an unavoidable name collision in dependent code. However, this is usually pretty unlikely, especially if you keep to your own namespace, so only an increase of the minor version number is required, in other words A.B MAY remain the same the new C MUST be greater than the old C. If, however, your added module name is taken from another package (e.g. when network-bytestring was merged into network) or is quite general (Data.Set or something similar) then the version increase SHOULD be major.

Therefore, we should only create a minor version `2.0.3`. Could you please update this file and the `.cabal` file?

Reply:
Note that this is also removing things previously exported by `Data.Csv.Parser.Megaparsec` and now only the new module exposes those (for example, `ConversionError`).

Perhaps those should continue to be exported (re-exported with this change) like before. Otherwise, I'm not sure how that works with the PVP.

Reply:
Oh, good catch @jpvillaisaza ! We should keep exporting `ConversionError` from `Data.Csv.Parser.Megaparsec`. @aemartinez37 Could you please make sure we keep exporting that type from the old module?